src/benchmarks/mod.rs

```diff
@@ -0,0 +1,2 @@
+#[cfg(feature = "benchmarks")]
+pub mod config;
```

Comment:
You can inline the contents of the `config` module into this module, no need for the extra nesting.